pyrefinebio/computational_result.py

```diff
@@ -0,0 +1,44 @@
+import pyrefinebio.common.annotation as prb_annotation
+import pyrefinebio.common.organism_index as prb_organism_index
```

Comment:
Why are annotation and organism_index nested in `common`?

Reply:
they're used by multiple classes.
organism_index is used by `computed_file` and `computational_result`
annotation is used by `computed_file`, `computational_result`, `experiment` and `sample`

I thought it made sense to separate because I wanted to keep the classes with `get` and `search` separate but I guess it doesn't really matter